Realm/Realm/Extensions/CollectionExtensions.cs

```diff
@@ -267,6 +267,42 @@ public static void Move<T>(this IList<T> list, int from, int to)
             return realmDictionary;
         }
 
+        /// <summary>
+        /// Converts a Realm-backed <see cref="IDictionary{String, T}"/> to a Realm-backed <see cref="IQueryable{T}"/>.
+        /// </summary>
+        /// <typeparam name="T">The type of the values contained in the dictionary.</typeparam>
+        /// <param name="dictionary">The dictionary of objects as obtained from a to-many relationship property.</param>
```

Comment:
I don't quite understand what "obtained from a to-many relationship property" means but I kept it here, not sure if that is correct

Reply:
Originally this was written for properties like `IList<Dogs`, where this actually represent a to-many relationship between a person and his dogs. I would say that this is still a to-many relationship. so it could make sense to keep it.